src/mlpack/core/tree/hollow_ball_bound.hpp

```diff
@@ -181,7 +181,8 @@ class HollowBallBound
   /**
    * Calculates minimum and maximum bound-to-point distance.
    *
-   * @param point Point to which the minimum and maximum distances are
+   * @param other Bound to check the condition.
+   * @param * Point to which the minimum and maximum distances are
```

Comment:
```suggestion
   * @param other Point to which the minimum and maximum distances are
```
Same here, I think this can be fixed.  I'm not sure I've caught everywhere this appears; I might have missed one...